envConfig/live.env

```
@@ -6,6 +6,10 @@ SIMORGH_ATI_BASE_URL=https://a1.api.bbc.co.uk/hit.xiti?
 SIMORGH_CONFIG_URL=https://config.api.bbci.co.uk/
 SIMORGH_INCLUDES_BASE_URL=https://www.bbc.com/ws/includes
 SIMORGH_INCLUDES_BASE_AMP_URL=https://news.files.bbci.co.uk
+SIMORGH_ADS_SCRIPT_TEST=https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap.js
+SIMORGH_ADS_SCRIPT_LEGACY_TEST=https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap-legacy.js
+SIMORGH_ADS_SCRIPT_LIVE=https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap.js
+SIMORGH_ADS_SCRIPT_LEGACY_LIVE=https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap-legacy.js
```

Are we supposed to use `/test/` for `SIMORGH_ADS_SCRIPT_LEGACY_LIVE`?
```suggestion
SIMORGH_ADS_SCRIPT_LEGACY_LIVE=https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap-legacy.js
```

---

Opps missed that one, fixed!